Public/css/style.css

```diff
@@ -22,6 +22,23 @@ body {
     padding-left: 40px;
 }
 
+/* Sponsors */
+
+.table {
+    border-collapse: unset;
+    vertical-align: middle;
+    text-align
+}
+
+.sponsor-img {
+    max-width: 100%;
+    height: auto;
+}
+
+th, td {
```

Comment:
This will force all table cells to be left-aligns so either needs to be more specific or be put into the global styles, not part of sponsors

Reply:
👍🏼